File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Bill/LoadGridBIL2016U02Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Bill
{
    public class LoadGridBIL2016U02Info
    {
        private String _hangmogCode;
        private String _hangmogName;
        private String _unit;
        private String _price;
        private String _quantity;
        private String _amount;
        private String _insurancePay;
        private String _patientPay;
        private String _discount;
        private String _deptReqCd;
        private String _deptReqNm;
        private String _doctorReqCd;
        private String _doctorReqNm;
        private String _deptExcCd;
        private String _deptExcNm;
        private String _doctorExcCd;
        private String _doctorExcNm;
        private String _orderGroupCd;
        private String _orderGroupNm;
        private String _orderDate;
        private String _actingDate;
        private String _discountReason;
        private String _checkYn;
        private String _fkocs1003;
        private String _amountPaid;
        private String _amountDebt;
        private String _fkout1001;
        private String _patientAddress;

        public String HangmogCode
        {
            get { return this._hangmogCode; }
            set { this._hangmogCode = value; }
        }

        public String HangmogName
        {
            get { return this._hangmogName; }
            set { this._hangmogName = value; }
        }

        public String Unit
        {
            get { return this._unit; }
            set { this._unit = value; }
        }

        public String Price
        {
            get { return this._price; }
            set { this._price = value; }
        }

        public String Quantity
        {
            get { return this._quantity; }
            set { this._quantity = value; }
        }

        public String Amount
        {
            get { return this._amount; }
            set { this._amount = value; }
        }

        public String InsurancePay
        {
            get { return this._insurancePay; }
            set { this._insurancePay = value; }
        }

        public String PatientPay
        {
            get { return this._patientPay; }
            set { this._patientPay = value; }
        }

        public String Discount
        {
            get { return this._discount; }
            set { this._discount = value; }
        }

        public String DeptReqCd
        {
            get { return this._deptReqCd; }
            set { this._deptReqCd = value; }
        }

        public String DeptReqNm
        {
            get { return this._deptReqNm; }
            set { this._deptReqNm = value; }
        }

        public String DoctorReqCd
        {
            get { return this._doctorReqCd; }
            set { this._doctorReqCd = value; }
        }

        public String DoctorReqNm
        {
            get { return this._doctorReqNm; }
            set { this._doctorReqNm = value; }
        }

        public String DeptExcCd
        {
            get { return this._deptExcCd; }
            set { this._deptExcCd = value; }
        }

        public String DeptExcNm
        {
            get { return this._deptExcNm; }
            set { this._deptExcNm = value; }
        }

        public String DoctorExcCd
        {
            get { return this._doctorExcCd; }
            set { this._doctorExcCd = value; }
        }

        public String DoctorExcNm
        {
            get { return this._doctorExcNm; }
            set { this._doctorExcNm = value; }
        }

        public String OrderGroupCd
        {
            get { return this._orderGroupCd; }
            set { this._orderGroupCd = value; }
        }

        public String OrderGroupNm
        {
            get { return this._orderGroupNm; }
            set { this._orderGroupNm = value; }
        }

        public String OrderDate
        {
            get { return this._orderDate; }
            set { this._orderDate = value; }
        }

        public String ActingDate
        {
            get { return this._actingDate; }
            set { this._actingDate = value; }
        }

        public String DiscountReason
        {
            get { return this._discountReason; }
            set { this._discountReason = value; }
        }

        public String CheckYn
        {
            get { return this._checkYn; }
            set { this._checkYn = value; }
        }

        public String Fkocs1003
        {
            get { return this._fkocs1003; }
            set { this._fkocs1003 = value; }
        }

        public String AmountPaid
        {
            get { return this._amountPaid; }
            set { this._amountPaid = value; }
        }

        public String AmountDebt
        {
            get { return this._amountDebt; }
            set { this._amountDebt = value; }
        }

        public String Fkout1001
        {
            get { return this._fkout1001; }
            set { this._fkout1001 = value; }
        }

        public String PatientAddress
        {
            get { return this._patientAddress; }
            set { this._patientAddress = value; }
        }

        public LoadGridBIL2016U02Info() { }

        public LoadGridBIL2016U02Info(String hangmogCode, String hangmogName, String unit, String price, String quantity, String amount, String insurancePay, String patientPay, String discount, String deptReqCd, String deptReqNm, String doctorReqCd, String doctorReqNm, String deptExcCd, String deptExcNm, String doctorExcCd, String doctorExcNm, String orderGroupCd, String orderGroupNm, String orderDate, String actingDate, String discountReason, String checkYn, String fkocs1003, String amountPaid, String amountDebt, String fkout1001, String patientAddress)
        {
            this._hangmogCode = hangmogCode;
            this._hangmogName = hangmogName;
            this._unit = unit;
            this._price = price;
            this._quantity = quantity;
            this._amount = amount;
            this._insurancePay = insurancePay;
            this._patientPay = patientPay;
            this._discount = discount;
            this._deptReqCd = deptReqCd;
            this._deptReqNm = deptReqNm;
            this._doctorReqCd = doctorReqCd;
            this._doctorReqNm = doctorReqNm;
            this._deptExcCd = deptExcCd;
            this._deptExcNm = deptExcNm;
            this._doctorExcCd = doctorExcCd;
            this._doctorExcNm = doctorExcNm;
            this._orderGroupCd = orderGroupCd;
            this._orderGroupNm = orderGroupNm;
            this._orderDate = orderDate;
            this._actingDate = actingDate;
            this._discountReason = discountReason;
            this._checkYn = checkYn;
            this._fkocs1003 = fkocs1003;
            this._amountPaid = amountPaid;
            this._amountDebt = amountDebt;
            this._fkout1001 = fkout1001;
            this._patientAddress = patientAddress;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/bill/LoadGridBIL2016U02Info.java
package nta.med.data.model.ihis.bill;

import java.math.BigDecimal;

public class LoadGridBIL2016U02Info {

	private String hangmogCode;
	private String hangmogName;
	private String unit;
	private BigDecimal price;
	private String quantity;
	private BigDecimal amount;
	private BigDecimal insurancePay;
	private BigDecimal patientPay;
	private BigDecimal discount;
	private String deptReqCd;
	private String deptReqNm;
	private String doctorReqCd;
	private String doctorReqNm;
	private String deptExcCd;
	private String deptExcNm;
	private String doctorExcCd;
	private String doctorExcNm;
	private String orderGroupCd;
	private String orderGroupNm;
	private String orderDate;
	private String actingDate;
	private String paymentName;
	private String paymentCode;
	private String paidName;
	private String discountReason;
	private Double fkocs1003;
	private BigDecimal totalDiscount;
	private String discountType;
	private String discountReasonMaster;
	private String revertType;
	private String revertComment;
	private BigDecimal amountPaid;
	private BigDecimal amountDebt;
	public LoadGridBIL2016U02Info(String hangmogCode, String hangmogName, String unit, BigDecimal price,
			String quantity, BigDecimal amount, BigDecimal insurancePay, BigDecimal patientPay, BigDecimal discount,
			String deptReqCd, String deptReqNm, String doctorReqCd, String doctorReqNm, String deptExcCd,
			String deptExcNm, String doctorExcCd, String doctorExcNm, String orderGroupCd, String orderGroupNm,
			String orderDate, String actingDate, String paymentName, String paymentCode, String paidName,
			String discountReason, Double fkocs1003, BigDecimal totalDiscount, String discountType,
			String discountReasonMaster, String revertType, String revertComment, BigDecimal amountPaid,
			BigDecimal amountDebt) {
		super();
		this.hangmogCode = hangmogCode;
		this.hangmogName = hangmogName;
		this.unit = unit;
		this.price = price;
		this.quantity = quantity;
		this.amount = amount;
		this.insurancePay = insurancePay;
		this.patientPay = patientPay;
		this.discount = discount;
		this.deptReqCd = deptReqCd;
		this.deptReqNm = deptReqNm;
		this.doctorReqCd = doctorReqCd;
		this.doctorReqNm = doctorReqNm;
		this.deptExcCd = deptExcCd;
		this.deptExcNm = deptExcNm;
		this.doctorExcCd = doctorExcCd;
		this.doctorExcNm = doctorExcNm;
		this.orderGroupCd = orderGroupCd;
		this.orderGroupNm = orderGroupNm;
		this.orderDate = orderDate;
		this.actingDate = actingDate;
		this.paymentName = paymentName;
		this.paymentCode = paymentCode;
		this.paidName = paidName;
		this.discountReason = discountReason;
		this.fkocs1003 = fkocs1003;
		this.totalDiscount = totalDiscount;
		this.discountType = discountType;
		this.discountReasonMaster = discountReasonMaster;
		this.revertType = revertType;
		this.revertComment = revertComment;
		this.amountPaid = amountPaid;
		this.amountDebt = amountDebt;
	}
	public String getHangmogCode() {
		return hangmogCode;
	}
	public void setHangmogCode(String hangmogCode) {
		this.hangmogCode = hangmogCode;
	}
	public String getHangmogName() {
		return hangmogName;
	}
	public void setHangmogName(String hangmogName) {
		this.hangmogName = hangmogName;
	}
	public String getUnit() {
		return unit;
	}
	public void setUnit(String unit) {
		this.unit = unit;
	}
	public BigDecimal getPrice() {
		return price;
	}
	public void setPrice(BigDecimal price) {
		this.price = price;
	}
	public String getQuantity() {
		return quantity;
	}
	public void setQuantity(String quantity) {
		this.quantity = quantity;
	}
	public BigDecimal getAmount() {
		return amount;
	}
	public void setAmount(BigDecimal amount) {
		this.amount = amount;
	}
	public BigDecimal getInsurancePay() {
		return insurancePay;
	}
	public void setInsurancePay(BigDecimal insurancePay) {
		this.insurancePay = insurancePay;
	}
	public BigDecimal getPatientPay() {
		return patientPay;
	}
	public void setPatientPay(BigDecimal patientPay) {
		this.patientPay = patientPay;
	}
	public BigDecimal getDiscount() {
		return discount;
	}
	public void setDiscount(BigDecimal discount) {
		this.discount = discount;
	}
	public String getDeptReqCd() {
		return deptReqCd;
	}
	public void setDeptReqCd(String deptReqCd) {
		this.deptReqCd = deptReqCd;
	}
	public String getDeptReqNm() {
		return deptReqNm;
	}
	public void setDeptReqNm(String deptReqNm) {
		this.deptReqNm = deptReqNm;
	}
	public String getDoctorReqCd() {
		return doctorReqCd;
	}
	public void setDoctorReqCd(String doctorReqCd) {
		this.doctorReqCd = doctorReqCd;
	}
	public String getDoctorReqNm() {
		return doctorReqNm;
	}
	public void setDoctorReqNm(String doctorReqNm) {
		this.doctorReqNm = doctorReqNm;
	}
	public String getDeptExcCd() {
		return deptExcCd;
	}
	public void setDeptExcCd(String deptExcCd) {
		this.deptExcCd = deptExcCd;
	}
	public String getDeptExcNm() {
		return deptExcNm;
	}
	public void setDeptExcNm(String deptExcNm) {
		this.deptExcNm = deptExcNm;
	}
	public String getDoctorExcCd() {
		return doctorExcCd;
	}
	public void setDoctorExcCd(String doctorExcCd) {
		this.doctorExcCd = doctorExcCd;
	}
	public String getDoctorExcNm() {
		return doctorExcNm;
	}
	public void setDoctorExcNm(String doctorExcNm) {
		this.doctorExcNm = doctorExcNm;
	}
	public String getOrderGroupCd() {
		return orderGroupCd;
	}
	public void setOrderGroupCd(String orderGroupCd) {
		this.orderGroupCd = orderGroupCd;
	}
	public String getOrderGroupNm() {
		return orderGroupNm;
	}
	public void setOrderGroupNm(String orderGroupNm) {
		this.orderGroupNm = orderGroupNm;
	}
	public String getOrderDate() {
		return orderDate;
	}
	public void setOrderDate(String orderDate) {
		this.orderDate = orderDate;
	}
	public String getActingDate() {
		return actingDate;
	}
	public void setActingDate(String actingDate) {
		this.actingDate = actingDate;
	}
	public String getPaymentName() {
		return paymentName;
	}
	public void setPaymentName(String paymentName) {
		this.paymentName = paymentName;
	}
	public String getPaymentCode() {
		return paymentCode;
	}
	public void setPaymentCode(String paymentCode) {
		this.paymentCode = paymentCode;
	}
	public String getPaidName() {
		return paidName;
	}
	public void setPaidName(String paidName) {
		this.paidName = paidName;
	}
	public String getDiscountReason() {
		return discountReason;
	}
	public void setDiscountReason(String discountReason) {
		this.discountReason = discountReason;
	}
	public Double getFkocs1003() {
		return fkocs1003;
	}
	public void setFkocs1003(Double fkocs1003) {
		this.fkocs1003 = fkocs1003;
	}
	public BigDecimal getTotalDiscount() {
		return totalDiscount;
	}
	public void setTotalDiscount(BigDecimal totalDiscount) {
		this.totalDiscount = totalDiscount;
	}
	public String getDiscountType() {
		return discountType;
	}
	public void setDiscountType(String discountType) {
		this.discountType = discountType;
	}
	public String getDiscountReasonMaster() {
		return discountReasonMaster;
	}
	public void setDiscountReasonMaster(String discountReasonMaster) {
		this.discountReasonMaster = discountReasonMaster;
	}
	public String getRevertType() {
		return revertType;
	}
	public void setRevertType(String revertType) {
		this.revertType = revertType;
	}
	public String getRevertComment() {
		return revertComment;
	}
	public void setRevertComment(String revertComment) {
		this.revertComment = revertComment;
	}
	public BigDecimal getAmountPaid() {
		return amountPaid;
	}
	public void setAmountPaid(BigDecimal amountPaid) {
		this.amountPaid = amountPaid;
	}
	public BigDecimal getAmountDebt() {
		return amountDebt;
	}
	public void setAmountDebt(BigDecimal amountDebt) {
		this.amountDebt = amountDebt;
	}
	
}
